dim/src/core.rs

```diff
@@ -95,115 +100,308 @@ pub async fn warp_core(
+        .route_service(
+            "/api/v1/host/admin_exists",
+            warp!(host::filters::admin_exists),
+        )
+        //     host::filters::admin_exists(conn.clone()),
+        //     /* library routes */
+        .route_service(
+            "/api/v1/library/*path",
+            warp::service({
+                routes::library::filters::library_get_self(conn.clone())
+                    //     routes::library::filters::library_get_self(conn.clone()),
+                    .or(routes::library::filters::get_all_unmatched_media(
+                        conn.clone(),
+                    ))
+                    //     routes::library::filters::get_all_unmatched_media(conn.clone()),
+                    .or(routes::library::filters::library_delete(conn.clone()))
+                    //     routes::library::filters::library_delete(conn.clone()),
+                    .or(routes::library::filters::library_get_self(conn.clone()))
+                    //     routes::library::filters::library_get_self(conn.clone()),
+                    .or(routes::library::filters::get_all_of_library(conn.clone()))
+                //     routes::library::filters::get_all_of_library(conn.clone()),
+            }),
+        )
+        .route_service(
+            "/api/v1/library",
+            warp::service(
+                library::filters::library_get(conn.clone())
+                    //     routes::library::filters::library_get(conn.clone()),
+                    .or(routes::library::filters::library_post(
+                        conn.clone(),
+                        event_tx.clone(),
+                    )), //     routes::library::filters::library_post(conn.clone(), event_tx.clone()),
+            ),
+        )
+        .route_service("/api/v1/dashboard", warp!(dashboard::filters::dashboard))
+        //     routes::dashboard::filters::dashboard(conn.clone(), rt.clone()),
+        .route_service(
+            "/api/v1/dashboard/banner",
+            warp!(dashboard::filters::banners),
+        )
+        //     routes::dashboard::filters::banners(conn.clone()),
+        .route_service("/api/v1/search", warp!(routes::general::filters::search))
+        //     routes::general::filters::search(conn.clone()),
+        .route_service(
+            "/api/v1/filebrowser/",
+            warp!(routes::general::filters::get_directory_structure),
+        )
+        .route_service(
+            "/api/v1/filebrowser/*path",
+            warp!(routes::general::filters::get_directory_structure),
+        )
+        //     routes::general::filters::get_directory_structure(conn.clone()),
+        .route_service("/images/*path", warp!(statik::filters::get_image))
+        //     routes::statik::filters::get_image(conn.clone()),
+        //     /* media routes */
+        .route_service(
+            "/api/v1/media/*path",
+            warp::service({
+                //     routes::media::filters::get_media_by_id(conn.clone()),
```

Comment:
Do we need all of these comments merged too?

Reply:
I'll remove them